src/simulation/Simulation.h

```diff
@@ -57,7 +57,7 @@ class Simulation : public LoadGenerator
 
     // returns true if all nodes have externalized
     // triggers and exception if a node externalized higher than num+maxSpread
-    bool haveAllExternalized(SequenceNumber num, uint32 maxSpread);
```

Comment:
what about adding LedgerNumber typedef?

Reply:
We could but that would require a code base sweep, I'll create an issue to discuss this - there are probably other types that we should typedef